index.html

```diff
@@ -1,23 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
+<head>
     <meta charset="utf-8">
     <meta name="description" content="Your description goes here">
     <meta name="keywords" content="one, two, three">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
 
-    <title>Blank Template</title>
+    <title>Relaxr Landing Page</title>
 
-    <!-- external CSS link -->
+  <!-- external CSS link -->
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
 
-  </head>
+</head>
 
-  <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+<body>
+  <div class= "container">  
+   <header>
+      <h1>Relaxr</h1>
+      <h2>Get piece of mind with a single tap</h2>
+        <div class="buttoncenter">
+          <button class="firstbutton">Get it Now</button>
+        </div>
+    </header>
+  </div>
+    <section class="container2 benefits">
+        <h2>Benefits</h2>
```

Comment:
Try removing the `margin-top` on these elements. Its creating white space between each group of content.